Makefile

```diff
@@ -56,20 +60,35 @@ tags:
 
 fetch_build:
 	curl -Lk -o $(ARCHIVE_PATH) $(BUILD_URL)/$(ARCHIVE_FILENAME)
+	if [ ! -z "$(DEPS_URL)" ]; then                                          \
+		set -e;                                                              \
```

Comment:
Should move this `set -e` up to between line 62 and 63.  In general, it is good practice to `set -e` at the very start of all pipelines (a command list separated by `;` is a _pipeline_ in the bourne shell) in a make recipe.